pkg/controllers/dynakube/logmonitoring/daemonset/volumes.go

```diff
@@ -130,6 +135,33 @@ func getIngestVolumes() []corev1.Volume {
 			VolumeSource: corev1.VolumeSource{
 				HostPath: &corev1.HostPathVolumeSource{
 					Path: containerLogsVolumePath,
+					Type: ptr.To(corev1.HostPathDirectory),
+				},
+			},
+		},
+	}
+}
+
+// getIngestVolumeMounts provides the VolumeMounts for the log folders that will be ingested by the logmonitoring
+func getJournalLogsVolumeMounts() []corev1.VolumeMount {
+	return []corev1.VolumeMount{
+		{
+			Name:      journalLogsVolumeName,
+			MountPath: journalLogsVolumePath,
+			ReadOnly:  true,
+		},
+	}
+}
+
+// getIngestVolumeMounts provides the VolumeMounts for the log folders that will be ingested by the logmonitoring
+func getJournalLogsVolumes() []corev1.Volume {
```

Comment:
technically, this is also an "ingest volume" as it is mounted so that log-monitoring can ingest its contents 😅 

also, the comments are copy-pasted 😄

Reply:
I moved it into the ingest volumes https://github.com/Dynatrace/dynatrace-operator/pull/4413/commits/226f5a9a17bf7b71a17fec9dd75a90d322124032